src/Entities/AccessTokenEntity.php

```diff
@@ -157,13 +158,20 @@ public function setRequestedClaims(array $requestedClaims): void
      */
     public function getState(): array
     {
+        $database = Database::getInstance();
```

Comment:
I don't understand why this is changing.  I think the PR should be focussed on things related to the processing chain.  I think if there are issues with the database unrelated to the processing chain then that should go in a different PR so the conversation can be more focussed.

Reply:
I am getting errors when i run the conformance tests, as a result i included the fixes in this PR. I will move them in a separate PR.

Reply:
moved to a different issue